jax/_src/numpy/lax_numpy.py

```diff
@@ -2324,10 +2325,13 @@ def _pad(array, pad_width, mode, constant_values):
     msg = "Unimplemented padding mode '{}' for np.pad."
     raise NotImplementedError(msg.format(mode))
 
+
 @_wraps(np.pad)
-def pad(array, pad_width, mode='constant', constant_values=0):
-  if isinstance(pad_width, list):
-    pad_width = tuple(pad_width)
+def pad(array, pad_width, mode="constant", constant_values=0):
+  if isinstance(pad_width, Sequence):
+    pad_width = tuple(  # new line
```

Comment:
Sometimes I've just done this via a trailing `#` with no text. (Note we don't use pyformat, so you can also just *not* use it here...)